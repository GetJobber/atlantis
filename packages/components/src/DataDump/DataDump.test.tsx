import React from "react";
import renderer from "react-test-renderer";
import { DataDump } from ".";

it("renders a DataDump", () => {
  const tree = renderer.create(<DataDump data={{ Foo: "bar" }} />).toJSON();
  expect(tree).toMatchSnapshot();
});

it("dumps some data with a label", () => {
  const tree = renderer
    .create(<DataDump label="ðŸ’©" data={{ Foo: "bar" }} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
