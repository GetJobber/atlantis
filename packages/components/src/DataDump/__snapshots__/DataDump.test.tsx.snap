// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dumps some data with a label 1`] = `
<div
  className="card accent yellow"
>
  <div
    className="padded base"
  >
    <details
      open={false}
    >
      <summary>
        <b
          className="base bold"
        >
          ðŸ’©
        </b>
      </summary>
      <pre>
        {
  "Foo": "bar"
}
      </pre>
    </details>
  </div>
</div>
`;

exports[`renders a DataDump 1`] = `
<div
  className="card accent yellow"
>
  <div
    className="padded base"
  >
    <details
      open={false}
    >
      <summary>
        <b
          className="base bold"
        >
          Raw Data
        </b>
      </summary>
      <pre>
        {
  "Foo": "bar"
}
      </pre>
    </details>
  </div>
</div>
`;
